taskcluster/translations_taskgraph/actions/train.py

```diff
@@ -58,6 +57,40 @@ def validate_pretrained_models(params):
             )
 
 
+def get_descendants(task_id, queue, res=None):
```

Comment:
If you feel like it, upstreaming this into https://github.com/taskcluster/taskgraph/blob/ec4a5ae2a0fd35e324d9531cb896023a5465eb79/src/taskgraph/util/taskcluster.py#L479 would be nice. Totally up to you.

Reply:
Yeah, it's a good idea. I was surprised there's `get_ancestors` method but no `get_descendants` one.

Reply:
They're added as needed. `get_ancestors` was added when I built the earlier continuation logic :)

Reply:
The issue is it's in `taskgraph` package, but I'm using `taskcluster` lib to get descendants (Queue etc.), so it doesn't belong there.